iotdb-core/datanode/src/test/java/org/apache/iotdb/db/pipe/extractor/PipeRealtimeExtractTest.java

```diff
@@ -102,12 +104,11 @@ public void tearDown() {
   public void testRealtimeExtractProcess() {
     // set up realtime extractor
 
-    try (PipeRealtimeDataRegionHybridExtractor extractor1 =
-            new PipeRealtimeDataRegionHybridExtractor();
+    try (PipeRealtimeDataRegionLogExtractor extractor1 = new PipeRealtimeDataRegionLogExtractor();
```

Comment:
Will this change the logic of this test?

Reply:
No. I think this is a really GOOD way to do tests :)